pages/index.md

```diff
@@ -6,38 +6,97 @@ PyCon UK is the annual gathering of the UK Python community and its friends from
 This year, we return to [Cardiff City Hall](http://www.cardiffcityhall.com/),
 from Thursday 26th to Monday 30th October.
 
-* * *
+**_Sold Out_**: we are completely sold out with over 700 people coming. We regret
+that we have no waiting list. If you are a sponsor or a speaker with a query about
+tickets please contact the [organising committee](/contact/)    .
 
-Tickets for the main conference have now sold out, and we're expecting over 700 people at the conference!
-(But there are still a handful of tickets left for the [children's day](https://hq.pyconuk.org/children/orders/new/).)
+We [still have tickets](https://hq.pyconuk.org/children/orders/new/) for the 
+[Children's Day](/education/).
 
 * * *
 
-There will be a heavyweight boxing match in Cardiff over the weekend of the
-conference, and so hotel accommodation in the city has been snapped up much
-more quickly this year than usual.  If you're coming to PyCon UK, *book your
-accommodation now*. Don’t delay any further. If you leave it too late, you may
-find that there is no more affordable accommodation available.  Please see the
-[accommodation page](/accommodation/) for more details.
+## Information for Attendees
 
-* * *
+You can review your talk details and the events and meals you've signed up for at 
+[your profile](https://hq.pyconuk.org/). You'll need to sign in using the email
+address with which you booked your ticket.
+
+### Registration & Schedule
+
+* The conference runs from 9am to 6.30pm each day.
+* Registration on each day opens at 8am but will be available throughout the day.
+* The [Schedule of talks](/schedule/) is available on the PyCon UK website.
+* [Monday](/schedule/#monday) is a day for Sprints: a chance to collaborate on
+  your own or someone else's Python project.
+* The [Conference Dinner](https://hq.pyconuk.org/dinners/conference-dinner/) 
+  is on Friday at the conference venue itself in the Lower Hall. 
+  The bar will be open from 6.30pm and dinner is served at 7.30pm.
+* The Contributors' Dinner is on Sunday at the 
+  [Clink Restaurant](http://theclinkcharity.org/the-clink-restaurants/cardiff-wales/) at [whatever time]
```

Comment:
Do we want to mention this on the front page, given that the majority of readers won’t be able to attend?

Reply:
I think we should include it, even though it's not relevant to most people, because a surprising number of people are bad at searching for emails with information.  But we should mention that it's invite-only, and tell speakers to get in touch if they've not received an invite.

Reply:
It's a bit 50/50 but my aim was for people to have a one-stop-shop of where to look. And there are more than 100 people going, I think. I didn't think it was too egregious.